src/config.ts

```diff
@@ -586,6 +586,29 @@ export const ARNS_NAMES_CACHE_TTL_SECONDS = +env.varOrDefault(
   `${60 * 60}`, // 1 hour
 );
 
+export const ARNS_NAMES_CACHE_CIRCUIT_BREAKER_TIMEOUT_MS = +env.varOrDefault(
+  'ARNS_NAMES_CACHE_CIRCUIT_BREAKER_TIMEOUT_MS',
+  `${10 * 1000}`, // 10 seconds
+);
+
+export const ARNS_NAMES_CACHE_CIRCUIT_BREAKER_ERROR_THRESHOLD_PERCENTAGE =
+  +env.varOrDefault(
+    'ARNS_NAMES_CACHE_CIRCUIT_BREAKER_ERROR_THRESHOLD_PERCENTAGE',
+    '30', // 30% failure limit before circuit breaker opens
```

Comment:
i.e 3/10 requests fail - we trigger the circuit breaker

Reply:
I was wondering if we should make the window smaller, but seems fine since 3 consecutive failures over 6 minutes could trigger this.